### DISCUSSION TITLE
Is the excalidraw plus sidebar included in this repo?

### DISCUSSION THREAD

**Discussion by mart3323** [excalidraw/excalidraw#8011] [Q&A] [answered]
I have some gripes with the current functionality that i know won't be addressed, so i would like to clone the repo and make myself a custom build that fixes them. However i would also like to use the collections and scene management that excalidraw+ provides.
Is this possible, or do i need to choose one-or-the-other?

**Comment by dwelle** [comment, marked as answer]
Hi. The Excalidraw+ is a separate app that's not open source, so you'd have to implement anything you need yourself.
